command/server/test-fixtures/tls_config_ok.hcl

```diff
@@ -0,0 +1,48 @@
+disable_cache = true
+disable_mlock = true
+
+ui = true
+
+listener "tcp" {
+    address = "127.0.0.1:1025"
```

Comment:
If I change this to 1024, I get "port already in use" when running the test. This is slightly concerning, since it may point to the fact that the listener may not be closed in the previous runs, but I'm hoping it's just because of the parallelism when running tests.